src/azure-cli/azure/cli/command_modules/hdinsight/_help.py

```diff
@@ -168,11 +168,23 @@
 
 helps['hdinsight update'] = """
 type: command
-short-summary: Update the tags or identity of the specified HDInsight cluster.
+short-summary: Update the tags or identity of the specified HDInsight cluster. Setting the identity property will override the existing identity configuration of the cluster.
 examples:
   - name: Update the tags.
     text: |-
         az hdinsight update --name MyCluster --resource-group rg --tags key=value
+  - name: Update manage identity with a UserAssigned msi.
```

Comment:
change a to single to align with the below multiple example

Reply:
Done